crates/bevy_sprite/src/render/mod.rs

```diff
@@ -294,7 +294,25 @@ impl SpecializedRenderPipeline for SpritePipeline {
                 topology: PrimitiveTopology::TriangleList,
                 strip_index_format: None,
             },
-            depth_stencil: None,
+            // Sprites are always alpha blended so they never need to write to depth.
```

Comment:
Is this because with alpha blending, we're not layering anything "on top", and therefore there's no need for the depth buffer to be written to?

In other words, the depth value would remain the same regardless of what we're drawing?

Reply:
Answered on discord to not bloat the comments too much